driver/razerchromacommon.c

```diff
@@ -1546,3 +1546,77 @@ struct razer_report razer_chroma_misc_get_scroll_smart_reel(void)
 
     return report;
 }
+
+/**
+ * Set LED mode for HyperPolling Wireless Dongle
+ * 1 = Connection Status
+ * 2 = Battery Status
+ * 3 = Battery Warning Only
+ */
+struct razer_report razer_chroma_misc_set_hyperpolling_wireless_dongle_indicator_led_mode(unsigned char mode)
+{
+    struct razer_report report = get_razer_report(0x07, 0x10, 0x01);
+
+    if(mode < 0x01 || mode > 0x03) {
+        mode = 0x01;
+    }
+
+    report.arguments[0] = mode;
+
+    return report;
+}
+
+/**
+ * Get LED mode for HyperPolling Wireless Dongle
+ */
+struct razer_report razer_chroma_misc_get_hyperpolling_wireless_dongle_indicator_led_mode(void)
+{
+    struct razer_report report = get_razer_report(0x07, 0x90, 0x01);
+
+    return report;
+}
+
+/**
+ * Set pairing mode for HyperPolling Wireless Dongle
+ */
+struct razer_report razer_chroma_misc_set_hyperpolling_wireless_dongle_pair(unsigned int pid, int step)
```

Comment:
Please use `unsigned short` everywhere for PId, that way you can also ignore overflows here

Reply:
Done!